packages/firebase_core_tizen/example/pubspec.lock

```diff
@@ -0,0 +1,194 @@
+# Generated by pub
```

Comment:
Is the file included by mistake??

Reply:
I though that having pubspec.lock in an app is a good thing. That's how it works with regular apps. But examples don't include it and I guess it's better without it. Removed.